### PR TITLE
Add gas price for send max

### PR DESCRIPTION
**Pull request by cehali**
# Summary

### CODE REVIEW

**Comment by cehali** [comment]
To test on mainnet

File: universal-login-commons/lib/core/constants/gas.ts [hunk 1]
```diff
@@ -5,6 +5,8 @@ export const DEFAULT_GAS_PRICE = 10000000000;
 
 export const DEFAULT_GAS_LIMIT = 200000;
 
+export const SEND_MAX_GAS_LIMIT = 95000;
```
Comment (xitronix) [inline universal-login-commons/lib/core/constants/gas.ts]: TRANSFER_GAS_LIMIT

**Comment by JustynaBroniszewska** [comment]
Needs rebase

**Comment by marekkirejczyk** [comment]
@cehali Conflicts

File: universal-login-commons/src/core/constants/gas.ts [hunk 2]
```diff
@@ -5,6 +5,8 @@ export const DEFAULT_GAS_PRICE = 10000000000;
 
 export const DEFAULT_GAS_LIMIT = 200000;
 
+export const SEND_MAX_GAS_LIMIT = 95000;
```
Comment (marekkirejczyk) [inline universal-login-commons/src/core/constants/gas.ts]: `SEND_TRANSACTION_GAS_LIMIT`

File: universal-login-sdk/src/core/services/TransferService.ts [hunk 3]
```diff
@@ -44,7 +44,7 @@ export class TransferService {
     if (gasToken !== ETHER_NATIVE_TOKEN.address) {
       return balance;
     }
-    const gasCostInWei = utils.bigNumberify(DEFAULT_GAS_LIMIT.toString()).mul(gasPrice);
+    const gasCostInWei = utils.bigNumberify(SEND_MAX_GAS_LIMIT.toString()).mul(gasPrice);
```
Comment (marekkirejczyk) [inline universal-login-sdk/src/core/services/TransferService.ts]: Also use in `transfer` function

**Pull request merged**